segmentation_models_pytorch/encoders/dpn.py

```diff
@@ -34,7 +34,7 @@
 
 class DPNEncoder(DPN, EncoderMixin):
     _is_torch_scriptable = False
-    _is_torch_exportable = False
+    _is_torch_exportable = True  # since torch 2.6.0
```

Comment:
Should this have an if-statement based on Python version?

Reply:
hmm.. this way, we should test other encoders for all the previous torch versions as well. `torch.export` is still in active development and it's recommended to use the latest torch, so I suppose a comment is enough unless we see an issue

Reply:
That's why I designed the tests to run on the newest and oldest version of all of our dependencies.

Reply:
Yes, I understand, but some features are introduced with the latest version of torch, while the main functionality works with the previous version. I'm not sure we should claim we are only compatible with PyTorch 2.6+ if only a very small subset of features fails. WDYT?

Reply:
I think it should be clearly stated in the docs/README that export/scripting/compilation has been tested and is expected to work with the latest versions and one may encounter issues with previous versions.

Reply:
We can support a wider range of PyTorch versions by using an if-statement on this one line of code. Alternatively, we can skip the export test for this specific model for certain versions of PyTorch.